packages/@azure/servicebus/data-plane/lib/clientEntityContext.ts

```diff
@@ -113,14 +118,23 @@ export namespace ClientEntityContext {
       entityPath: entityPath,
       requestResponseLockedMessages: new ConcurrentExpiringMap<string>(),
       isSessionEnabled: !!options.isSessionEnabled,
-      messageSessions: {}
+      messageSessions: {},
+      expiredMessageSessions: {}
     };
 
     (entityContext as ClientEntityContext).sessionManager = new SessionManager(
       entityContext as ClientEntityContext
     );
 
     (entityContext as ClientEntityContext).getReceiver = (name: string, sessionId?: string) => {
+      if (sessionId != undefined && entityContext.expiredMessageSessions[sessionId] === true) {
```

Comment:
This can be simplified to 
`if (sessionId && entityContext.expiredMessageSessions[sessionId])`